src/main/java/org/jvnet/libpam/AuthenticatedUser.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2009, Sun Microsystems, Inc.
```

Comment:
I presume this is incorrect, since this class was not made in 2009 by Sun though it looks like it's primarily pulled from UnixUser, which was.

Suggestions on what I should update it to while I'm changing things?